build.gradle

```diff
@@ -11,18 +11,23 @@ apply plugin: 'project-report' // adds dependencyReport and htmlDependencyReport
 
 sourceCompatibility = 1.8
 
-// Defaults for variables
+boolean isTestEnv = project.hasProperty("testEnv")
+
+// Defaults for development variables
 run.environment("DB_DRIVER", "sqlserver")
 run.environment("ANET_DB_SERVER", "localhost")
-run.environment("ANET_DB_NAME", "testAnet")
-run.environment("ANET_DB_USERNAME", "anetUser")
-run.environment("ANET_DB_PASSWORD", "Test-P@ssw0rd")
+run.environment("ANET_DB_NAME", isTestEnv ? "testAnet" : "devAnet")
+run.environment("ANET_DB_USERNAME", isTestEnv ? "anetTestUser" : "anetUser")
```

Comment:
Since we're changing the defaults here anyway, why not change `"anetUser"` to `"anetDevUser"`?